website/screens/components/date-input/code/examples/uncontrolled.ts

```diff
@@ -0,0 +1,22 @@
+import { DxcDateInput, DxcInset } from "@dxc-technology/halstack-react";
+
+const code = `() => {
```

Comment:
I feel that it can also make the example more confusing since what we want to demonstrate is the use of an uncontrolled date as neither other example adds any interaction with other components. 
Also we might be implying that this is how we want forms to be implemented.

Reply:
This example was agreed upon a few months ago and it's the same for every input/select/textarea component. The main goal here is not to showcase how you should do a form, instead, it teaches the new users how to extract the input value of an uncontrolled component using the `ref` prop. In this case, it is necessary to use another component since you can achieve that demostration with just an input.